app/com/linkedin/drelephant/tuning/IPSOConstants.java

```diff
@@ -0,0 +1,34 @@
+package com.linkedin.drelephant.tuning;
+
+public class IPSOConstants {
```

Comment:
Can't we call this class just Constants instead of IPSOConstants? This way it can be leveraged by entire auto tuning as there is nothing specific to IPSO in this.